docs/tutorials/join-dal-baker/run-baker.md

```diff
@@ -1,35 +1,105 @@
 ---
-title: "Step 5: Run an Octez baking daemon on Weeklynet"
-authors: Tezos core developers
+title: "Step 4: Run an Octez baking daemon"
+authors: Tezos core developers, Tim McMackin
 last_update:
-  date: 23 January 2024
+  date: 12 August 2024
 ---
 
-The baking daemon is launched almost as usual, the only difference is that we use the `--dal-node http://127.0.0.1:10732` option to tell it to connect to the DAL node that we just launched in the previous step.
+To run a baking daemon that connects to the DAL, start it as usual and pass the URL to your DAL node to it:
 
+1. In a new terminal window, run this command:
 
-```bash
-octez-baker-alpha run with local node "$HOME/.tezos-node" my_baker --liquidity-baking-toggle-vote on --adaptive-issuance-vote on --dal-node http://127.0.0.1:10732 >> "$HOME/octez-baker.log" 2>&1
-```
+   ```bash
+   octez-baker-PsParisC run with local node "$HOME/.tezos-node" my_baker --liquidity-baking-toggle-vote pass --adaptive-issuance-vote on --dal-node http://127.0.0.1:10732 >> "$HOME/octez-baker.log" 2>&1
+   ```
 
-We can check that the DAL is now subscribed to the relevant topics by retrying the following RPC call, which should now return all the topics of the form `{"slot_index":<index>,"pkh":"<ADDRESS OF OUR BAKER>"}` where `index` varies between `0` included and the number of slot indexes (`32` on Weeklynet) excluded:
+1. In another terminal window, install the `curl` program by running this command:
 
-```bash
-curl http://localhost:10732/p2p/gossipsub/topics
-```
+   ```bash
+   sudo apk add curl
+   ```
+
+1. In the same terminal window, run this command:
```

Comment:
I would put this in a dedicated section like: `Check the DAL node is well configured` that says:

- When the bakers starts, it registers to the DAL node with the profile (we could give it earlier actually) that will tell which topics to track.
- Check your DAL node still have some connections

I will try to ensure there is a producer but it is not the case at the moment, so we cannot guarantee there is some connections in each topics, but what is important is that the node is still connected to the bootstrap nodes.

Reply:
There are three types of profiles:

- Operator (if you are a rollup operator)
- Observer (if you are a rollup observer)
- Attester (if you are an attester)

When nothing is mentioned on the CLI, it is none of the three above, so there is no topics.

When the baker starts and a DAL node is provided, it patches the DAL node profile (via an RPC) to take into account the profile associated to the baker.

Reply:
Working on clarifying the modes/profiles here: https://gitlab.com/tezos/tezos/-/merge_requests/14591